src/chrome/content/rules/STA-Travel.com.sg.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	For other coverage, see STA-Travel.com.xml
+
+
+	Non-functional subdomains:
+		- $host		(t)
+		- www		(i)
```

Comment:
The chain seems to be okay, but I get 404 and HTTP redirects.

Reply:
I mislabeled, fixed.